src/services/expenses.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+let nextExpenseId = 1;
+
+let expenses = [];
+
+const getExpensesData = () => {
+  return expenses;
+};
+
+const clearExpensesArray = () => {
+  expenses = [];
+};
+
+function postExpense(body) {
+  const expense = {
+    id: nextExpenseId++,
+    ...body,
+  };
+
+  expenses.push(expense);
+
+  return expense;
+}
+
+function getExpenses(query) {
+  const { userId, category, from, to } = query;
+  const numberFromDate = new Date(from).getTime();
+  const numberToDate = new Date(to).getTime();
```

Comment:
All this logic should be in the controller, move it.

In controllers should be all logic, in services just data and methods to get or update this data.